exporter/tanzuobservabilityexporter/exporter.go

```diff
@@ -0,0 +1,164 @@
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package tanzuobservabilityexporter
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"net/url"
+	"strconv"
+
+	"github.com/google/uuid"
+	"github.com/wavefronthq/wavefront-sdk-go/senders"
+	"go.opentelemetry.io/collector/config"
+	"go.opentelemetry.io/collector/consumer/consumererror"
+	"go.opentelemetry.io/collector/consumer/pdata"
+	"go.uber.org/zap"
+)
+
+const (
+	LabelApplication   = "application"
+	LabelError         = "error"
+	LabelEventName     = "name"
+	LabelService       = "service"
+	LabelSpanKind      = "span.kind"
+	LabelStatusMessage = "status.message"
+	LabelStatusCode    = "status.code"
+)
+
+// Sender Interface for sending tracing spans to Tanzu Observability
+type Sender interface {
```

Comment:
Does this need to be public?

Reply:
Nope! We made this private and renamed to `spanSender` in #3422.